v2.0/operational-faqs.md

```diff
@@ -38,26 +38,53 @@ Like most databases, CockroachDB caches the most recently accessed data in memor
 
-To truncate timeseries data sooner, you can change the `timeseries.resolution_10s.storage_duration` cluster setting to an [`INTERVAL`](interval.html) value less than `720h0m0s` (30 days). For example, to truncate timeseries data after 15 days, you would execute the following [`SET CLUSTER SETTING`](set-cluster-setting.html) command:
-
-{% include copy-clipboard.html %}
-~~~ sql
-> SET CLUSTER SETTING timeseries.resolution_10s.storage_duration = '360h0m0s';
-~~~
-
-{% include copy-clipboard.html %}
-~~~ sql
-> SHOW CLUSTER SETTING timeseries.resolution_10s.storage_duration;
-~~~
-
-~~~
-+--------------------------------------------+
-| timeseries.resolution_10s.storage_duration |
-+--------------------------------------------+
-| 360h                                       |
-+--------------------------------------------+
-(1 row)
-~~~
+## Can I reduce the amount of timeseries data stored by CockroachDB?
+
+Yes. CockroachDB stores timeseries data for the last 30 days by default, but you can reduce the interval for storage or disable timeseries storage entirely.
+
+- To reduce the interval for storage of timeseries data, change the `timeseries.resolution_10s.storage_duration` cluster setting to an [`INTERVAL`](interval.html) value less than `720h0m0s` (30 days). For example, to store timeseries day for the last 15 days, run the following [`SET CLUSTER SETTING`](set-cluster-setting.html) command:
+
+    {% include copy-clipboard.html %}
+    ~~~ sql
+    > SET CLUSTER SETTING timeseries.resolution_10s.storage_duration = '360h0m0s';
+    ~~~
+
+    {% include copy-clipboard.html %}
+    ~~~ sql
+    > SHOW CLUSTER SETTING timeseries.resolution_10s.storage_duration;
+    ~~~
+
+    ~~~
+    +--------------------------------------------+
+    | timeseries.resolution_10s.storage_duration |
+    +--------------------------------------------+
+    | 360h                                       |
+    +--------------------------------------------+
+    (1 row)
+    ~~~
+
+- To disable the storage of timeseries data entirely, run the following command:
+
+    {% include copy-clipboard.html %}
+    ~~~ sql
+    > SET CLUSTER SETTING timeseries.storage.enabled = false;
+    ~~~
+
+    {% include copy-clipboard.html %}
+    ~~~ sql
+    > SHOW CLUSTER SETTING timeseries.storage.enabled;
+    ~~~
+
+    ~~~
+    +----------------------------+
+    | timeseries.storage.enabled |
+    +----------------------------+
+    | false                      |
+    +----------------------------+
+    (1 row)
+    ~~~
+
+{{site.data.alerts.callout_info}}After adjusting either of these cluster settings, it can take up to 24 hours for timeseries data to be deleted and for the change to be reflected in Admin UI metrics.{{site.data.alerts.end}}
```

Comment:
@dianasaur323, @mrtracy, @vilterp, this sound ok?

Reply:
Based on seeing this in the code: https://github.com/cockroachdb/cockroach/blob/3dbe1dae8e734a27df53591f882efcd1543de9cd/pkg/storage/ts_maintenance_queue.go#L31-L33

Now that I look again though, I see that that's a _per-replica_ timer, and replica timers could be staggered. So maybe a little more accurate to say that it could take up to 24 hours for it to be deleted _across all replicas_.

I'm not sure if that adds a lot though; what you have now seems good. Thoughts @mrtracy?